### ISSUE TITLE
Drop source installation instructions in favor of developer manual

### ISSUE THREAD

**Issue by ralf401** [zammad/zammad-documentation#581]
In case of reworking the instructions, at least the following points should be checked for Ubuntu Server 24.04:

- Add command for adding user to `rvm` group
- Investigate which command has to be executed as which user
- Investigate if there is another possibility than add `sudo` group to user `zammad`.
- Check why adding rails environment variables doesn't work from a different user than `zammad` (even with sudo)
- TBD

**Comment by YetAnotherGerrit** [comment]
I agree that from source installation is something rarely used in production. But we need to make sure to not close the doors to open-source development enthusiasts. Maybe we can join it somehow with the "how to contribute" manual, so someone who wants to contribute as a developer can get a dev env up and running?

**Comment by ralf401** [comment]
> But we need to make sure to not close the doors to open-source development enthusiasts. Maybe we can join it somehow with the "how to contribute" manual, so someone who wants to contribute as a developer can get a dev env up and running?

I don't think we are closing any doors by dropping it. There is a [developer manual ](https://github.com/zammad/zammad/blob/develop/doc/developer_manual/index.md) on Github where people can find detailed installation instructions (see [How to set up a development environment](https://github.com/zammad/zammad/blob/develop/doc/developer_manual/development_environment/how-to-set-up-a-development-environment.md)). This development guide is even linked in the [zammad repo README.md](https://github.com/zammad/zammad?tab=readme-ov-file#developer-manual).

So my suggestion is to temporarily link to this Github developer manual from the source installation page and add it to the contribution section. After the next release we can remove the souce installation page from the documentation.

Any objections?

**Comment by YetAnotherGerrit** [comment]
I talked with several stakeholders, no veto from my side. Thank you for taking care!
